marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.tsx

```diff
@@ -43,11 +49,75 @@ const ResultsTabularView = (props) => {
         return columns;
     }
 
-    const tableHeaders = tableHeaderRender(selectedTableColumns);
+
+    const updatedtableHeader = () => {
+        let header = tableHeaderRender(selectedTableColumns);
+        let detailView = {
+            dataIndex: 'detailView',
+            key: 'key',
+            title: 'Detail View'
+        }
+        header.push(detailView);
+        return header;
+    }
+
+    const tableHeaders = updatedtableHeader();
+
 
     const tableDataRender = (item) => {
         let dataObj = {};
-        for (let subItem of item) {
+        if (!Array.isArray(item) && item.hasOwnProperty('entityProperties')) {
+
+            let itemEntityName: string[] = [];
+            let itemEntityProperties: any[] = [];
+            let entityDef: any = {};
+            let primaryKeyValue: string = '';
+            let primaryKeys: string[] = [];
+            let entityTitle: string[] = [];
+            if (item.extracted.hasOwnProperty('content') && item.extracted.content[1]) {
```

Comment:
@xnikhil08 @rahulvudutala We don't want the UI doing any of this work. The backend should do all of this and present exactly the data that the UI needs. 

Now that we have a search transform in place that is defining entityProperties, it should be simple for the backend to add any other data points that the UI needs. It looks like the backend should provide the URI (which is in each search result already) and an "identifier" value, which is absolutely something we want the backend to calculate. 

Maybe we need a quick meeting on this after the scrum today?